quinn-udp/src/unix.rs

```diff
@@ -138,12 +145,19 @@ impl UdpSocketState {
             // kernel's path MTU guess, but actually disabling fragmentation requires this too. See
             // __ip6_append_data in ip6_output.c.
             // Set `may_fragment` to `true` if this option is not supported on the platform.
-            may_fragment |= !set_socket_option_supported(
-                &*io,
-                libc::IPPROTO_IPV6,
-                libc::IPV6_DONTFRAG,
-                OPTION_ON,
-            )?;
+            #[cfg(not(target_os = "openbsd"))]
+            {
+                may_fragment |= !set_socket_option_supported(
+                    &*io,
+                    libc::IPPROTO_IPV6,
+                    libc::IPV6_DONTFRAG,
+                    OPTION_ON,
+                )?;
+            }
+            #[cfg(target_os = "openbsd")]
+            {
+                may_fragment |= true;
+            }
```

Comment:
Why are we skipping this? OpenBSD seems to define the symbol:

https://github.com/openbsd/src/blob/35659bf2453d7eeb12fd24089cf668ca13361c18/sys/netinet6/in6.h#L336

Reply:
The libc crate does not have this symbol, I would prefer to have something sub-optimal working that can later be improved than wait until this is fully supported.

Reply:
We don't need to wait for `libc` to define the constant for us; it's just a number. Let's define the constant ourselves, ideally with a TODO pointing to a libc issue or PR.

Reply:
Cool, I've updated this now to just define the constant.  All still works.